framework/controller/controller.gotext

```diff
@@ -69,6 +69,19 @@ func ({{$action.Short}} *{{ $.Pascal }}{{$action.Pascal}}Action) handler(httpRes
 			JSON: response.Status(400).Set("Content-Type", "application/json").JSON(map[string]string{"error": err.Error()}),
 		}
 	}
+	// validate input
+	{{- if (index $action.Params 0).HasValidate }}
+	// single input struct has validate method
+	if err := in.Validate(); err != nil {
+		return &response.Format{
+			{{- if ne $action.Method "GET" }}
+			HTML: response.Status(http.StatusSeeOther).RedirectBack(httpRequest.URL.Path),
```

Comment:
The big question is going to be what happens with the error message? Right now if a form fails, there's no feedback sent back to the user. 

[Flash messages](https://api.rubyonrails.org/classes/ActionDispatch/Flash.html) are typically the solution to this where you store a value in a cookie that gets removed after it's pulled out. I think the ultimate solution is to have the ability to add [custom flash messages via the session](https://laravel.com/docs/9.x/session#flash-data), but if there's any temporary solution to get this working where you set the cookie, pull it out and pass to the view, I'd be open to pursuing that. Here's what I've thought about so far with sessions: https://github.com/livebud/bud/discussions/56.

Otherwise maybe we only enable this for JSON? Does your use case fit with that limitation?

Reply:
Yea, I think flash messages should be the behavior here. I'm thinking to add some other stuff to this PR but then put in on pause until we have sessions and something like flash messages.

Reply:
Cool, sounds good! I'm going to get back to feature development very soon. Just trying to wrap up generator caching to allow for more expensive generators to be able to run (e.g. generators that call `go run`)